### DISCUSSION TITLE
[New Feature] AA missiles are unreliable

### DISCUSSION THREAD

**Discussion by CCTiberius** [Vinifera-Developers/Vinifera#1505] [Feature Requests]
### Description:
Rockets/missiles fired at moving aircraft often deal no damage to them when they detonate. It seems random when it happens, although a high speed on the aircraft seems to make it worse

### Conditions:
A weapon using any sort of rocket/missile projectile

### Steps To Reproduce:
1. Send a bunch of aircraft to attack in an area where there is rocket-based anti-air like SAMs

### Expected Behaviour:
The rockets would pursue the aircraft for a set amount of time or distance, and if they hit, they will always deal damage

### Actual Behaviour:
The rockets will seem to detonate but the aircraft will randomly receive no damage. Also the distance a rocket will pursue an aircraft seems entirely random

**Comment by Crimsonum** [comment]
This is part of a hardcoded behavior where missiles (projectiles with ROT>1) often detonate prematurely when the target is moving away from them. This applies to both ground and air targets. You can mitigate it somewhat with a high `Arm=` time, but it doesn't prevent the issue entirely.

**Comment by CCTiberius** [comment]
I think it would be very nice to be able to not only entirely prevent it, but set the fix to apply to AA only. Missing moving targets can be a good feature in ground combat I think, but the sheer randomness of it in regards to AA is just frustrating.

**Comment by Rampastring** [comment]
I think this might not really be a bug, but it would be nice to be able to have more control over when the projectile explodes.
